dotnet/src/SemanticKernel.Abstractions/AI/PromptExecutionSettings.cs

```diff
@@ -27,6 +27,22 @@ public class PromptExecutionSettings
     /// </remarks>
     public static string DefaultServiceId => "default";
 
+    /// <summary>
+    /// Service identifier.
+    /// This identifies the service these settings are configured for e.g., openai, ollama, huggingface, etc.
```

Comment:
The service id is an arbitrary identifier so include examples like `azure_openai_eastus, ...`